config/api_config.go

```diff
@@ -78,3 +78,8 @@ func (cfg *APIConfig) Load(v *viper.Viper) error {
 	cfg.readIn(v)
 	return v.Unmarshal(&cfg)
 }
+
+// APIEndpoint returns the full URL for a given endpoint
+func APIEndpoint(endpoint string) string {
```

Comment:
Yeah I figured that would be the case, but I thought I'd best be safe than sorry.